cmd/mattermost/commands/command_test.go

```diff
@@ -117,11 +117,11 @@ func TestCreateCommand(t *testing.T) {
 			cmds, _ := th.SystemAdminClient.ListCommands(team.Id, true)
 
 			if testCase.ExpectedErr == "" {
-				if len(cmds) == 0 || cmds[0].Trigger != "testcmd" {
-					t.Fatal("Failed to create command")
-				}
+				require.NotEqual(t, len(cmds), 0, "Failed to create command")
```

Comment:
You can use  `require.Len` here.

Reply:
@hanzei doesn't `require.Len` test if the length is equal to some number but in this case, we're testing if the length is not equal to 0

Reply:
I see. I thought we had expectations on length of the list, but it seams like we don't

Reply:
We could probably use `require.NotEmpty`.